package.json

```diff
@@ -54,7 +56,7 @@
 	"devDependencies": {
 		"@govflanders/vl-ui-util": "3.9.1",
 		"@govflanders/vl-ui-progress-bar": "3.9.1",
```

Comment:
Deze mogen dan wellicht hier weg?

Reply:
Klopt, zal ik verwijderen op de #master branch, kan voorlopig geen kwaad dus nieuwe release is niet nodig.